.github/workflows/pr-preview.yml

```diff
@@ -14,7 +14,8 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  S3_BUCKET: ghgc-custom-interfaces-develop
+  S3_BUCKET: ghgc-dev-custom-interfaces
```

Comment:
can we please keep using the same bucket? (it follows the same convention for staging and production buckets too, so don't want to be confused later on)
just do the work needed to update cloudfront to have access to the bucket